keyboards/keychron/q0/q0.c

```diff
@@ -16,6 +16,20 @@
 
 #include "quantum.h"
 
+#ifdef ENCODER_ENABLE
```

Comment:
It might work to have this at `/keyboards/keychron/keychron.c`.

Reply:
As above